tests/appsec/test_processor.py

```diff
@@ -147,6 +147,21 @@ def test_appsec_cookies_no_collection_snapshot(tracer):
     assert "triggers" in json.loads(span.get_tag("_dd.appsec.json"))
 
 
+@snapshot(include_tracer=True)
+def test_appsec_body_no_collection_snapshot(tracer):
```

Comment:
I see the snapshot test but I've not seen the snapshot/*.json file, you should commit them :+1:

Reply:
Done :+1: